packages/sanity/src/structure/comments/src/components/reactions/CommentReactionsMenu.tsx

```diff
@@ -83,9 +82,12 @@ export function CommentReactionsMenu(props: CommentReactionsMenuProps) {
             onClick={handleOptionClick}
             padding={2}
             role="menuitem"
-            selected={selected}
-            text={emoji}
-          />
+            tabIndex={-1}
```

Comment:
Added to prevent slight focus flickering when pressing TAB when you have an existing emoji focused in this menu